src/helpers/organisations/filterExtremes.ts

```diff
@@ -6,9 +7,12 @@ export function filterExtremes(simulationRecaps: SimulationRecap[]) {
   return simulationRecaps.filter((simulationRecap) => {
     // Remove simulations with too high values
     if (
-      [simulationRecap.bilan, Object.values(simulationRecap.categories)].some(
-        (value) => (value as number) > MAX_VALUE
-      )
+      [
+        simulationRecap.computedResults[carboneMetric].bilan,
+        Object.values(
```

Comment:
Shouldn't you spread here ?

Reply:
I think so yes. I guess it never worked. But I also don't understand why we filter on the backend AND the frontend.

@bjlaa